test/tests/relationship_managers/ghost_boundary/gold/test_out.csv

```diff
@@ -0,0 +1,3 @@
+time,num_rms
+0,0
+1,2
```

Comment:
shouild it not be 1?

Reply:
There's another relationship manager from the base `SideUserObjectClass`, `GhostLowerDElems`